configuration/defaults/staging/configuration.yaml

```diff
@@ -0,0 +1,269 @@
+# part of the system for agreeing on and synchronising protocol updates.
+LastKnownBlockVersion-Major = 1;
+LastKnownBlockVersion-Minor = 0;
+LastKnownBlockVersion-Alt = 0;
+
+# In the Byron era some software versions are also published on the chain.
+# We do this only for Byron compatibility now.
+ApplicationName = "cardano-sl";
+ApplicationVersion = 1;
+
+
+##### Logging configuration #####
+
+# The node can run in either the SimpleView or LiveView. The SimpleView just
+# uses standard output, optionally with log output. The LiveView is a text
+# console with a live view of various node metrics.
+ViewMode: SimpleView
+
+# Enble or disable logging overall
+TurnOnLogging: True
+
+# Enable the collection of various OS metrics such as memory and CPU use.
+# These metrics can be directed to the logs or monitoring backends.
+TurnOnLogMetrics: True
+
+# Global logging severity filter. Messages must have at least this severity to
+# pass. Typical values would be Warning, Notice, Info or Debug.
+minSeverity: Notice
+
+# Log items can be rendered with more or less verbose detail.
+# The verbosity can be: MinimalVerbosity, NormalVerbosity
+TracingVerbosity: NormalVerbosity
+
+# The system supports a number of backends for logging and monitoring.
+# This setting lists the the backends that will be available to use in the
+# configuration below. The logging backend is called Katip. Also enable the EKG
+# backend if you want to use the EKG or Prometheus monitoring interfaces.
+# You can also enable trace forwarder backend if node should send the metrics
+# to an external process.
+setupBackends:
+  - KatipBK
+# - EKGViewBK
+# - TraceForwarderBK
+
+# This specifies the default backends that trace output is sent to if it
+# is not specifically configured to be sent to other backends.
+defaultBackends:
+  - KatipBK
+
+# EKG is a simple metrics monitoring system. Uncomment the following to listen
+# on the given local port and point your web browser to http://localhost:12788/
+# for a live view. The same URL can also serve JSON output.
+# hasEKG: 12788
+
+# The Prometheus monitoring system can also be used. Uncomment the following
+# to listen on the given port
+# hasPrometheus: 12789
+
+# For the Katip logging backend we must set up outputs (called scribes)
+# The available types of scribe are:
+#   FileSK for files
+#   StdoutSK/StdoutSK for stdout/stderr
+#   JournalSK for systemd's journal system
+#   DevNullSK
+# The scribe output format can be ScText or ScJson. Log rotation settings can
+# be specified in the defaults below or overidden on a per-scribe basis here.
+setupScribes:
+  - scKind: FileSK
+    scName: "logs/mainnet.log"
+    scFormat: ScText
+
+  - scKind: StdoutSK
+    scName: stdout
+    scFormat: ScText
+    scRotation: null
+
+# For the Katip logging backend this specifies the default scribes that trace
+# output is sent to if it is not configured to be sent to other scribes.
+defaultScribes:
+  - - FileSK
+    - "logs/mainnet.log"
```

Comment:
This should be `staging.log`

Reply:
The Staging and Testnet files from this PR are identical with the ones from mainnet so they are not correct. Unfortunately, we/I don't know what should be the specific changes for Staging/Testnet.

Reply:
this should be fixed by the latest commit